winpcapy/winpcapy.py

```diff
@@ -109,6 +109,17 @@ def run(self, callback=None, limit=0):
         self._callback = callback
         # Run loop with callback wrapper
         wtypes.pcap_loop(self._handle, limit, self._callback_wrapper, None)
+    def send(self, buf):
```

Comment:
Very nice contribution.  👍 
Let's work at it a little to polish it, so it would produce the best results. 
Please see comments below.